weldx/schemas/weldx.bam.de/weldx/core/transformations/coordinate_system_hierarchy-1.0.0.yaml

```diff
@@ -10,114 +10,59 @@ description: |
-          transformation: !<asdf://weldx.bam.de/weldx/tags/core/transformations/local_coordinate_system-1.0.0>
-            time: !<asdf://weldx.bam.de/weldx/tags/time/timedeltaindex-1.0.0>
-              values: !core/ndarray-1.0.0
-                data: [0, 29000000000]
-                datatype: int64
-                shape: [2]
-              start: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-1.0.0> {value: P0DT0H0M0S}
-              end: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-1.0.0> {value: P0DT0H0M29S}
-              min: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-1.0.0> {value: P0DT0H0M0S}
-              max: !<asdf://weldx.bam.de/weldx/tags/time/timedelta-1.0.0> {value: P0DT0H0M29S}
-            orientations: !<asdf://weldx.bam.de/weldx/tags/core/variable-1.0.0>
-              name: orientations
-              dimensions: [c, v]
-              dtype: <f8
-              data: !core/ndarray-1.0.0
-                data:
-                - [1.0, 0.0, 0.0]
-                - [0.0, -1.0, -1.2246467991473532e-16]
-                - [0.0, 1.2246467991473532e-16, -1.0]
-                datatype: float64
-                shape: [3, 3]
-            coordinates: !<asdf://weldx.bam.de/weldx/tags/core/variable-1.0.0>
-              name: coordinates
-              dimensions: [time, c]
-              dtype: <f8
-              data: !core/ndarray-1.0.0
-                data:
-                - [5.0, 0.0, 2.0]
-                - [295.0, 0.0, 2.0]
-                datatype: float64
-                shape: [2, 3]
-        subsystem_names: []
+
+definitions:
+  coordinate_system_node:
+    allOf:
+      - oneOf:
+          - tag: "asdf://weldx.bam.de/weldx/tags/core/graph/di_node-1.0.0"
+      - type: object
+        properties:
+          # attributes:
+          #   type: object
+          #   properties:
+          #     data:
+          #       type: array
+          edges:
+            type: array
+            items:
+              $ref: "#/definitions/coordinate_transformation_edge"
+        required: [ attributes ]
+
+  coordinate_transformation_edge:
+    allOf:
+    - oneOf:
+      - tag: "asdf://weldx.bam.de/weldx/tags/core/graph/di_edge-1.0.0"
+    - type: object
+      properties:
+        attributes:
+          type: object
+          properties:
+            lcs:
+              tag: "asdf://weldx.bam.de/weldx/tags/core/transformations/local_coordinate_system-1.*"
+          required: [lcs]
+        target_node:
+          $ref: "#/definitions/coordinate_system_node"
+      required: [attributes]
+
 
 properties:
   name:
     type: string
 
-  root_system_name:
-    description: |
-      Name of the root coordinate system
-    type: string
-
   reference_time:
-    description: |
-      A timestamp that serves as reference point for all attached coordinate systems.
     oneOf:
       - tag: "asdf://weldx.bam.de/weldx/tags/time/timestamp-1.*"
-      - tag: "tag:weldx.bam.de:weldx/time/timestamp-1.*" # legacy_tag
```

Comment:
So I left them away, if I got you righi here.

Reply:
since the layout is now completely different and there are (cant be) any legacy files with this structure we can stick to the new tags
It would be nice however to still be able to read the old weldx files (and convert them afterwards)